src/pages/ImportData.test.js

```diff
@@ -0,0 +1,27 @@
+import { isProtected } from './ImportData';
+
+describe('isProtected', () => {
+  const protectedUrls = [
+    { url: 'https://prod.anvil.gi.ucsc.edu/file', format: 'pfb' },
+    { url: 'https://anvilproject.org/file2', format: 'PFB' },
+    { url: 'https://dev.anvil.gi.ucsc.edu/manifest/files', format: 'pFb' },
+    { url: 'https://gen3.biodatacatalyst.nhlbi.nih.gov/explorer', format: 'PFB' },
+  ];
+  const nonProtectedUrls = [
+    { url: 'https://nonanvil.site.org/file', format: 'pfb' },
+    { url: 'https://prod.anvil.gi.ucsc.edu/file', format: 'entitiesJson' },
+    { url: 'https://google.com/file.pfb', format: 'PFB' },
+    { url: 'https://nonanvil.site.org/file', format: 'tdrexport' },
+    { url: 'https://prod.anvil.gi.ucsc.edu/file', format: 'snapshot' },
+    { url: 'https://anvilproject.org/', format: 'catalog' },
+    { url: 'https://gen3.biodatacatalyst.nhlbi.nih.gov/explorer', format: 'snapShot' },
+  ];
+
+  test.each(protectedUrls)('%o should  be protected', ({ url, format }) => {
```

Comment:
I've never used `each()` myself but it looks like the most common way this is used written `it.each([ ... cases ... ])()`, not `test.each(something))()`.
```
$ find . -name \*.test.\?s | xargs grep test.each | wc
       2       4     180
$ find . -name \*.test.\?s | xargs grep it.each | wc
      70     420    7956
$
```

Reply:
Ah my bad.  That's an easy enough fix